src/library/scala/collection/parallel/immutable/ParRange.scala

```diff
@@ -107,6 +107,7 @@ self =>
     }
   }
 
+  override def toString = s"Par${range.toString}"
```

Comment:
A thing of beauty. It could actually be `s"Par$range"`. Does that look more like a riddle?